cv_pom/cv_pom_driver.py

```diff
@@ -168,7 +170,11 @@ def element(self, query: dict) -> CVPOMDriverElement:
         Returns:
             CVPOMDriverElement
         """
-        ocr = "text" in query
+        ocr = None
+        if "text" in query:
+            ocr = {'paragraph': False}
+            if self.kwargs and self.kwargs['ocr']:
+                ocr = self.kwargs['ocr']
```

Comment:
I'm not understanding here.
If there is text in query, you assign ocr value, but if there is also ocr in the arguments, you replace the value for the arguments, then could be more clear to use:

```suggestion
            if "text" in query:
                 ocr = {'paragraph': False}
            elif self.kwargs and self.kwargs['ocr']:
                ocr = self.kwargs['ocr']
```

The same happens in lines 191-195

Reply:
Hey, but no, this is not good, because we only want to use OCR in case the user ask for text in the query, meaning, if their query uses only a label, using OCR is just going to slow down the decoding of the image

Reply:
Understood, thanks!